automation_tools/__init__.py

```diff
@@ -1007,6 +1012,8 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         **installer_options
     )
 
+    execute(setup_scap_client, host=host)
```

Comment:
Every time execute is called on `product_install` it should specify `host=host`. It could be a machine when called with `-H root@server.com` or can be a VM when creating one and the `-H root@provisioning-host.com` will be the provisioning machine, after creating the VM the host will be updated to run on the VM instead of the provisioning host.